dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go

```diff
@@ -280,6 +280,11 @@ func generateTarball(hadoopClients []string) error {
 	for _, file := range []string{"bin/hive-checker.sh", "bin/mapreduce-checker.sh", "bin/spark-checker.sh"} {
 		replace(filepath.Join("integration/checker", file), "target/alluxio-checker-${VERSION}-jar-with-dependencies.jar", "alluxio-checker-${VERSION}.jar")
 	}
+	// Update the kubernetes image tags
+	for _, file := range []string{"alluxio-master.yaml.template", "alluxio-worker.yaml.template"} {
+		replace(filepath.Join("integration/kubernetes", file), "alluxio/alluxio:latest", fmt.Sprintf("alluxio/alluxio:%v", version))
+	}
+	replace("integration/kubernetes/alluxio-fuse.yaml.template", "alluxio/alluxio-fuse:latest", fmt.Sprintf("alluxio/alluxio-fuse:%v", version))
```

Comment:
Should this be done in the release branch similar to how we update for the DockerFile?

Reply:
not sure i follow. do you mean manually updating the ARG_TARBALL in the release branch? 
and, why do you prefer the alternate approach? to me it seems like all generated tarballs would benefit from having the correct tag

Reply:
Yeah updating the version in the release branch. The difference would be if they check out the tag/release branch directly, they still get the correct version.

Reply:
i see
- our docs will point to using the specs from the docker image. i imagine the common flow to not use the github specs, so i was trying to save the manual effort of updating the release branch
- the best of both would be an automated script to generate the tag/release branch but i'm not aware of any?

let me know what you think @calvinjia

Reply:
There already is kind of a checklist for updating strings when making a new tag, so I think we could just add this to it.

Reply:
sgtm

Reply:
reverted